share/picongpu/examples/FieldAbsorberTest/cmakeFlags

```diff
@@ -30,68 +30,68 @@
-#flags[21]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2'"
+#flags[20]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2'"
+#flags[21]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2'"
 
 # Convolutional PML in 2D, default strength, no stretching
-#flags[22]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=6;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-#flags[23]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-#flags[24]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=12;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-#flags[25]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[22]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=6;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[23]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[24]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=12;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[25]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
 
 # Convolutional PML in 2D, default strength, stretching
-#flags[26]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-#flags[27]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML< CurrentInterpolation >;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[26]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+#flags[27]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=YeePML<CurrentInterpolation>;-DPARAM_PML_SIZE=16;-DPARAM_PML_KAPPA_MAX=10.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
 
 # Unixaxial PML in 3D, default strength, no stretching
```

Comment:
Would you mind correcting `Unixaxial` to `Uniaxial` right with this PR? (4 times)

Reply:
I will fix it in separate PR, this PR passed now the CI.

Reply:
PR is open: #3528